language_names/manual.py

```diff
@@ -0,0 +1,33 @@
+import sqlite3
+import textwrap
+import csv
+
```

Comment:
Please sort imports and remove one of the lines here.

Reply:
Okay I will sort imports but I am not sure which line you want me to remove here

Reply:
There were two empty lines. Should be only one I think.

Reply:
No it give me error on running travis test. There must be 2 lines in between.

Reply:
Very well